contracts/application/PackagedApp.sol

```diff
@@ -31,17 +32,18 @@ contract PackagedApp is BaseApp {
   }
 
   /**
-   * @dev Changes the application's current version
-   * @dev Contract implementations for the given version must already be registered in the package
-   * @param newVersion Name of the new version
+   * @dev Sets the current version of the application.
+   * Contract implementations for the given version must have been registered in the package.
```

Comment:
"must have been" reads as "probably were", revert to "already be" or "need to be".. Or maybe "have previously been"? But it also allows that reading :-\

Reply:
reverted.